binderhub/app.py

```diff
@@ -251,6 +251,51 @@ def _add_slash(self, proposal):
         """
     )
 
+    use_as_federation_portal = Bool(False, config=True,
+        help="""configure the current binder to also act as a federation portal.
+
+        This settings should be used only on mybinder.org deployment or if you
+        have multiple internal private deployment of binderhubs inside a closed
+        network. It exposes a couple of extra endpoints to binderhub for remote
+        services to register themselves. Once register for a specific user, the
+        binder portal will redirect users to these endpoints instead of running
+        them locally.
+        """
+    )
+
+    default_binders_list = List([], config=True,
```

Comment:
do we need this to end in `_list`? `default_binders` would work for me or `default_remote_binders`?